typescript/prepare.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+npm install --no-package-lock
```

Comment:
yarn?

Reply:
This script is for the client to run so I don't know if they have `yarn` installed.
I prefer to use `npm` and just don't create `package-lock.json`.